### PR TITLE
Added missing styles to listbox, combobox, and menu

### PR DESCRIPTION
**Pull request by agliga**
## Description

Bold styles for ds6 were missing for listbox menu and combobox
Proper spacing for the tickbox was missing too
Also an overflow visible was added which forced listbox to infinitely scroll

## References

Fixes ebay/ebayui-core#814 and ebay/ebayui-core#811

## Screenshots

image

### CODE REVIEW

File: src/less/combobox/base/combobox.less [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (seangates) [inline src/less/combobox/base/combobox.less]: Errant new line.

File: src/less/listbox-button/base/listbox-button.less [hunk 2]
```diff
@@ -50,6 +49,17 @@ svg.listbox-button__status {
 }
 
 // background icon
+.listbox-button__status {
```
Comment (ianmcburnie) [inline src/less/listbox-button/base/listbox-button.less]: If I remember correctly, these elements are flex children and so these first and last child margin rules were replaced by me with a simple 'margin-left: auto'. If they are not flex children (perhaps they only are in certain dropdowns), then ignore my comment.

Comment (ianmcburnie) [reply src/less/listbox-button/base/listbox-button.less]: Here: https://github.com/eBay/skin/commit/28c696b1479f82da9163f26364e6377d512defb0#diff-4babd5c8f7e99f21b79c4125ef2e5c3bR65

Comment (agliga) [reply src/less/listbox-button/base/listbox-button.less]: @ianmcburnie The thing is when I don't remove these styles it looks like this: 
image
And the styles are applied as so, 
image

If that looks fine, then I'm fine with it too.

Comment (seangates) [reply src/less/listbox-button/base/listbox-button.less]: Nope. We need the 8px margin (or padding) on the left of the check mark. Something to make sure there's always space.

Comment (ianmcburnie) [reply src/less/listbox-button/base/listbox-button.less]: @agliga Gotcha. So long as this doesn't revert menu and combobox, and is listbox specific, then I'm fine with it. At some point I'll take a look and see why they are all different.

**Pull request merged**